_includes/footer.html

```diff
@@ -1,12 +1,63 @@
-<footer class="usa-footer usa-footer-medium usa-section background-gray-dark" role="contentinfo">
+<footer class="usa-footer usa-section background-medium" role="contentinfo">
   <div class="usa-grid">
-    <h2>Drop us a line.</h2>
-    <p>To share your thoughts, email us anytime at <a href="mailto:{{ site.footer.email }}">{{ site.footer.email }}</a>.</p>
+    <div class="footer-logo-links">
+    {% include icons/oes.svg %}
```

Comment:
we received feedback from GSA branding about switching the colors - so I used y'all's guidance to edit the GSA logo svg to meet the original branding standards (and change the background color of the footer). Thank you for your help - I'm learning!!